utils/src/main/kotlin/Utils.kt

```diff
@@ -64,6 +66,24 @@ val ortDataDirectory by lazy {
  */
 var printStackTrace = false
 
+/**
+ * Archives a given [input directory][inputDir] into the given []ZIP file][zipFile].
```

Comment:
s/Archives/Archive, s/[]/[/

Maybe we should make more clear that not `inputDir` itself is archived, but only its contents (i.e. the ZIP does not contain a single top-level directory). And we should also document `prefix` and mention the return value. Like:

> Archive the contents of [inputDir], omitting common [VCS_DIRECTORIES], to [zipFile] where an optional [prefix] is added to each file. Return a [Result] wrapping the [zipFile] on success, or an exception of failure.